content/news/2024-02-17-bevy-0.13/index.md

```diff
@@ -1392,17 +1392,17 @@ The new asset processing pipeline, using the `LoadTransformAndSave` `Process` im
 4. An `AssetSaver` takes a bevy `Asset` and converts it back into byte data.
 5. An `AssetWriter` then writes the asset byte data back to the asset source.
 
-In addition to having better code reusability, this change encorages writing `AssetSaver`s for various common asset types, which could be used to add runtime asset saving functionality to the `AssetServer`.
+In addition to having better code reusability, this change encourages writing `AssetSaver`s for various common asset types, which could be used to add runtime asset saving functionality to the `AssetServer`.
 
 The previous `LoadAndSave` `Process` implementation still exists, as there are some cases where an asset transformation step is unnecessary, such as when saving assets into a compressed format.
 
 See the [Asset Processing Example](<https://github.com/bevyengine/bevy/blob/main/examples/asset/processing/asset_processing.rs>) for a more detailed look into how to use `LoadTransformAndSave` to process a custom asset.
 
-## Entity Optimizations
+## Entity Optimisations
```

Comment:
"Optimizations" and "Optimisations" are both valid English, just one is American and the other British. In previous blog posts, "optimizations" was used almost exclusively. I'd rather be consistent with past work.